backend-comparison/benches/binary.rs

```diff
@@ -15,6 +15,14 @@ impl<B: Backend, const D: usize> Benchmark for BinaryBenchmark<B, D> {
         "Binary Ops".into()
     }
 
+    fn operation(&self) -> Option<String> {
+        Some("binary".to_string())
+    }
+
+    fn shapes(&self) -> Option<Vec<String>> {
+        Some(vec![format!("{:?}", self.shape.dims)])
+    }
```

Comment:
Shapes are supposed to be numbers, so maybe `Vec<Vec<usize>>` is better. Also, I don't think we need an option of vec; an empty vec will do the job if no shapes are provided.